.ci/deploy.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -eo pipefail # Exit with nonzero exit code if anything fails
+
+SOURCE_BRANCH="master"
+TARGET_BRANCH="gh-pages"
+
+# Pull requests and commits to other branches shouldn't try to deploy, just build to verify
+if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_BRANCH" != "$SOURCE_BRANCH" ]; then
+    echo "Skipping deploy; just doing a build."
+    # Don't waste token on PR
+    unset GITHUB_TOKEN
```

Comment:
the PR wont get an env var on Travis, or any other sane system (Netlify isnt sane/safe), so this is useless on Travis.

but more importantly, this is useless.  If the token is available here, a malicious person could modify this script to emit it to the log , or send it via email to themselves, or anything.

However this isnt such a large problem if the token in GITHUB_TOKEN has no scopes which allow greater access than the malicious person could obtain themselves.  The worst they can do is abuse the token so that GitHub cancels it, breaking the builds.

Reply:
Yeah, didn't realize Travis remove all env var on PR.

The token has no scope, unless you want to do deployment using Travis (which in this case we don't).